flexmeasures_openweathermap/sensor_specs.py

```diff
@@ -0,0 +1,37 @@
+from datetime import timedelta
+
+
+"""
+This maps sensor specs which we can use in FlexMeasures to OWM labels.
+Note: Sensor names we use in FM need to be unique per weather station.
+At the moment, we only extract from OWM hourly data.
+"""
+
+
+weather_seasonality = {
+    "daily_seasonality": True,
+    "weekly_seasonality": False,
+    "yearly_seasonality": True,
+}
+
+
+owm_to_sensor_map = dict(
+    temp={
+        "name": "temperature",
+        "unit": "°C",
+        "event_resolution": timedelta(minutes=60),
+        "seasonality": weather_seasonality,
+    },
+    wind_speed={
+        "name": "wind_speed",
+        "unit": "m/s",
+        "event_resolution": timedelta(minutes=60),
+        "seasonality": weather_seasonality,
+    },
+    clouds={
+        "name": "radiation",
```

Comment:
There is code in FlexMeasures using these names (also "wind_speed"), e.g. here in `model_spec`factory`:

```
elif sensor.generic_asset.generic_asset_type.name in [
        "wind_speed",
        "radiation",
    ]:
        # Values cannot be negative and are often zero
        return BoxCoxTransformation(lambda2=0.1)
```

You want me to change them as well? And/or should we make a ticket to spec normalisation attriibutes in the sensor data model?

Reply:
@Flix6x *ping*

Reply:
> You want me to change them as well?

Yes, please. To something like `sensor.name in ["wind speed", "irradiance"]`.

> And/or should we make a ticket to spec normalisation attributes in the sensor data model?

Yes, that kind of explicitness would be preferable imo. I made https://github.com/FlexMeasures/flexmeasures/issues/370.

Reply:
> Yes, please. To something like sensor.name in ["wind speed", "irradiance"].

I see that the terms "wind_speed" and "radiation" are, next to tests, also in use in other parts of the code, like `ui/views/analytics`, `data/models/assets.py`, `data/models/weather.py`, `ui/utils/view_utils.py`, `api/common/utils/validators.py`.

I'll switch them all out and see if tests still pass.

Reply:
There are some problems in tests, because we made FM turn every " " in weather sensors and weather sensor types in to "_".
And the weather sensor type name is used in the entity address `fm0` scheme ... which might be a reason to disallow spaces in sensor names, but I feel it's outdated.
Our new sensor class is not doing this.